docs_nnx/guides/gemma.ipynb

```diff
@@ -212,7 +237,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Finally, build a sampler on top of your model and your tokenizer."
```

Comment:
Added the source code with the docstring.

"Build a Flax NNX [`Sampler`](https://github.com/google/flax/blob/main/examples/gemma/sampler.py) on top of your model and tokenizer with the right parameter shapes."